SeQuant/core/space.hpp

```diff
@@ -340,7 +346,12 @@ class IndexSpace {
     const auto attr = Attr(type, qns);
     assert(attr.is_valid());
     if (attr == Attr::null()) return null_instance();
-    if (!instance_exists(attr)) throw bad_attr();
+    if (!instance_exists(attr)) {
+      std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+      throw bad_attr(converter.to_bytes(L"Request to non-existing space: " +
```

Comment:
Maybe
```suggestion
      throw bad_attr(converter.to_bytes(L"Request to non-registered space: " +
```
is preferable?

Reply:
To my knowledge they have first deprecated and now apparently removed the `codecvt` header in the C++ standard but without offering any alternative. That means one would have to write custom platform-depend code to perform these conversions - see also https://stackoverflow.com/a/42946556

Maybe another reason to reconsider #96? :D